app/grants/views.py

```diff
@@ -472,6 +479,12 @@ def subscription_cancel(request, grant_id, grant_slug, subscription_id):
         subscription.cancel_tx_id = request.POST.get('sub_cancel_tx_id', '')
         subscription.active = False
         subscription.save()
+        activity_kwargs = {
```

Comment:
i see 
```
Activity.objects.create(**activity_kwargs)
```
s 

for `killed_grant_contribution`
and for `new_grant_contribution`

but i dont see anywhere activity object creation code for `killed_grant` or `new_grant`. 

are you going to add those?

Reply:
Only for `new_grant_subscription`, `new_grant_contribution` and `killed_grant_contribution` because these are the ones that are going to appear in the user's profile. The others are out of scope